src/mame/heathkit/h19.cpp

```diff
@@ -148,7 +148,7 @@ TIMER_CALLBACK_MEMBER(h19_state::key_click_off)
 
 TIMER_CALLBACK_MEMBER(h19_state::bell_off)
 {
-	m_keyclickactive = false;
+	m_bellactive = false;
 
 	if (!m_keyclickactive && !m_bellactive)
```

Comment:
Arguably this `if` would make more sense in a specific function helper instead of repeating the logic twice (cfr. `key_click_off`).

Reply:
Agree that would be better. There was a single if check prior to the PR 9788.